src/index.css

```diff
@@ -1,6 +1,6 @@
 @layer framework, overrides;
 
-@import url("bootstrap.css") layer(framework);
+@import url('bootstrap.css') layer(framework);
 
 @layer overrides {
```

Comment:
The bottom is very tight to the edge of the screen. Maybe add a bit of space on html or body?

Reply:
fixed